share/completions/krita.fish

```diff
@@ -0,0 +1,55 @@
+function __krita_append_image_format_value -a value current_token
+    string replace -r '^' -- $current_token $value |
+        string replace -r '$' ','
```

Comment:
Okay, I didn't get what this was supposed to do at first, but this is just

```fish
string join , $value $current_token
```

I would remove this function and just do it in-line in __krita_complete_image_format.